### PR TITLE
Issue #488: upgrade to checkstyle 10.12.1

### LINKED ISSUE #488: upgrade to checkstyle 10.12.1
https://checkstyle.org/releasenotes.html#Release_10.12.1

### PR DESCRIPTION
**Pull request by muhlba91**
fixes #488

### CODE REVIEW

**Review by romani — APPROVED** [review]
Please proceed with self merge for such simple updates

**Comment by romani** [comment]
Usually you update readme table also

**Comment by muhlba91** [comment]
Thank you for the note; I somehow missed it... fixed now :)

File: README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ Compatibility matrix from checkstyle team:
 
 | Checkstyle Plugin | Sonar min | Sonar max | Checkstyle | Jdk |
 |-------------------|-----------|-----------|------------|-----|
+| 10.12.1           | 9.0       | 10.0+     | 10.12.0    | 11  |
```
Comment (romani) [inline README.md:24]: 10.12..1 for checkstyle

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: README.md (hunk 1 above)
Comment (muhlba91) [reply README.md]: resolved - thank you.

**Review by romani — APPROVED** [review]

**Pull request merged**